src/index.html

```diff
@@ -31,8 +33,6 @@ <h4 class="text-light">LEADERBOARD <span class="text text-success font-weight-bo
     </div>
 </div>
 
-
```

Comment:
- [ ] `README.md` file: To make your documentation looks more professional and descriptive, I think it would be better if you update some of your `sub-sections` of the `Getting Started` section, such as of  how to `access` the cloned directory, `install` dependencies, `run build` for production, `run/use` the app,... To help you, I suggest, you refer **[this gist file.](https://gist.github.com/tresorsawasawa/cc5e9a9e5dd20c0179ae98eca3b1ae96)**